### ISSUE TITLE
Add unit tests

### ISSUE THREAD

**Issue by paradox41** [paradox41/angular-chartist.js#24]

**Comment by ilanbiala** [comment]
Yeah right now https://github.com/paradox41/angular-chartist.js/blob/master/test/angular-chartist.spec.js#L16 isn't really testing much...I've tried testing Chartist on my own but I get an error:
```
TypeError: 'undefined' is not a function (evaluating 'function resizeListener(){
	        this.update();
	      }.bind(this)')
```
Thoughts?

**Comment by paradox41** [comment]
I really just have to think about the best way to test this directive. I've been trying to find other directives that are well tested, if you know of any, please feel free to post them here.

**Comment by ilanbiala** [comment]
https://github.com/angular-ui/bootstrap has lots of good tests in the src/ directory under each ui component.

**Comment by paradox41** [comment]
Hey! So I actually get tests working setup a little better (they actually run now). Do you have a branch where you are working on tests? I'd like to assist you if at all possible so we aren't duplicating our efforts.

**Comment by ilanbiala** [comment]
https://github.com/HTHS/hths-lunch/tree/develop/test/client/unit/directives is where I have some directives. I'm using a little bit different of a syntax though.

**Comment by paradox41** [comment]
Closed somewhat via 0a88a8bd1dbf831061ffdd31469f6a0b90edb563

Will add more as time goes on.
